Intersect (Core)/Config/PacketOptions.cs

```diff
@@ -6,24 +6,86 @@ namespace Intersect.Config
     public class PacketOptions
```

Comment:
This file is never added to the options file, so is never actually made configurable.

Reply:
PacketOptions are a part of SecurityOptions which are in the Options file

An argument could be made for the new options being in a different subclass than packet flooding, maybe packet timing instead?

![Settings](https://www.ascensiongamedev.com/resources/filehost/508d292ab2723fd944356018b9102488.png)

Reply:
That's odd, mine refused to generate those options so I figured they weren't linked properly.

I'll have to take another look when I get near a computer again.

Reply:
Maybe we just rename the PacketFlooding header to PacketSanitization?

Reply:
Per the convo in Discord with @lodicolo I renamed that section to just be 'Packets"